server/src/internalClusterTest/java/org/elasticsearch/action/admin/cluster/node/tasks/TasksIT.java

```diff
@@ -593,24 +616,8 @@ private <T> void waitForCompletionTestCase(boolean storeResult, Function<TaskId,
             // Wait for the task to start
             assertBusy(() -> clusterAdmin().prepareGetTask(taskId).get());
 
-            // Register listeners so we can be sure the waiting started
-            CountDownLatch waitForWaitingToStart = new CountDownLatch(1);
-            for (TransportService transportService : internalCluster().getInstances(TransportService.class)) {
-                ((MockTaskManager) transportService.getTaskManager()).addListener(new MockTaskManagerListener() {
-                    @Override
-                    public void onTaskUnregistered(Task task) {
-                        waitForWaitingToStart.countDown();
-                    }
-                });
-            }
```

Comment:
I am moving this to the test case specific to the list.
This is irrelevant to get task as it may or may not fork depending on the node receiving request and actually running the TEST_TASK

Reply:
Good point.

Reply:
I should have clarified it, this waits for the list task to start on all nodes, not the test task, so it is specific to the list test case.